CSV/boxes.csv

```diff
@@ -8,6 +8,7 @@ bidx,Box Index,ISO-Partial
 bloc,Base location and purchase location for license acquisition,DECE
 bmdm,Buffer Model Description,JPXS
 bpcc,Bits per component,JPEG2000
+brob,Brotli-compressed box. First four characters of the box content are a box type. Remaining bytes are compressed according to IETF RFC 7932,JPEG XL
```

Comment:
I find the description of the box a bit long. This CSV is not meant to be describing the structure of the box. Can we remove "First four ... RFC7932"?

Reply:
Sure, feel free to adjust the level of granularity to what is suitable.

Reply:
Simply saying "Brotli-compressed box" would be good.

Reply:
OK, sorry for the delay, updated it.